tofu.html

```diff
@@ -19,6 +19,8 @@
 <body>
 	<p>ABC 123</p>
 	<p class="tofu-fallback">ABC 123</p>
+	<p>🌶️</p>
```

Comment:
Confusing to have two pairs of text blocks here. Instead, just append the pepper to the end of "ABC 123" above.